service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/Message.java

```diff
@@ -210,9 +210,8 @@ public Message(byte[] byteArray)
      * Important: If a string is passed, the HttpBatch.SerializeAsString is set to true,
      * and the internal byte representation is serialized as UTF-8,
      * with HttpBatch.Encoding set to UTF-8.
-     * @throws UnsupportedEncodingException This exception is thrown if unsupported encoding used
      */
-    public Message(String string) throws UnsupportedEncodingException
```

Comment:
Users do use this API when sending c2d messages, so this would be a breaking change